doc/cephfs/fs-volumes.rst

```diff
@@ -45,9 +45,29 @@ Create a volume using::
 
     $ ceph fs volume create <vol_name> [<placement>]
 
-This creates a CephFS file system and its data and metadata pools. It also tries
-to create MDSes for the filesystem using the enabled ceph-mgr orchestrator
-module  (see :doc:`/mgr/orchestrator`) , e.g., rook.
+This creates a CephFS file system and its data and metadata pools. It can also
+try to create MDSes for the filesystem using the enabled ceph-mgr orchestrator
+module (see :doc:`/mgr/orchestrator`), e.g. rook.
+
+<vol_name> is the volume name (an arbitrary string), and
+
+<placement> is an optional string signifying which hosts should have NFS Ganesha
```

Comment:
can we simply link to :ref:`orchestrator-cli-placement-spec` here?

Reply:
@sebastian-philipp We need a place that documents **only** placement strings and doesn't mention YAML files. But @varshar16 says YAML file support is coming for `ceph fs` and `ceph fs`. Once that is implemented, then this section could be replaced with your link?

Reply:
Not for `ceph fs`, only for `ceph fs nfs` we can have YAML file support. The nfs yaml will not be same as `orch` yaml file and this yaml file will need to be transformed according to `orch` yaml file.

Example:
fs nfs yaml file
```
cluster type:
cluster id:
placement:
config:
```

Only the placement part has to be similar to `orch` yaml file.